src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1526,12 +1533,20 @@ k
         cc.save();
         cc.translate(this.canvas.panX, this.canvas.panY);
 
+        // TODO: Remove the jQuery in favor of _each()!
         $.each(airport.terrain || [], (elevation, terrainLevel) => {
+            if (elevation < 1000) {
+                console.warn(`${airport.icao}.geojson contains 'terrain' at or` +
+                    ' below sea level, which is not supported!');
+                // within `$.each()`, this return acts like `continue;`
+                return;
+            }
+
             max_elevation = Math.max(max_elevation, elevation);
-            const color = `rgba(${UiController.terrain.COLOR[elevation]}, `;
+            const color = `hsla(${UiController.terrain.COLOR[elevation]}`;
```

Comment:
why the move to `hsla`?  `hsla` tends to be a non-standard color format for the front-end.  it works, it's just not used very often in my experience.